src/main/java/uk/nhs/hee/tis/revalidation/connection/service/MasterElasticSearchService.java

```diff
@@ -51,18 +39,16 @@
 @Service
 public class MasterElasticSearchService {
 
-  private static final Logger LOG = LoggerFactory.getLogger(MasterElasticSearchService.class);
-
   @Autowired
   MasterElasticSearchRepository masterElasticSearchRepository;
 
   @Autowired
   ConnectionInfoMapper connectionInfoMapper;
 
-  @Value("${spring.elasticsearch.rest.uris}")
-  private URL ES_URL;
+  @Autowired
+  ElasticsearchRestTemplate elasticsearchTemplate;
```

Comment:
Constructor injection is preferred over Autowiring

Reply:
Used constructor injection instead.